{"uuid":"938563c8-66e0-48c4-9ae7-c24b256aaafb","historyId":"3910b8d4be3e6e6f6bc3579728cfb1bd","fullName":"automation.RegistroUsuarioTests.iniciarSesionConCredencialesValidas","labels":[{"name":"package","value":"automation.RegistroUsuarioTests"},{"name":"testClass","value":"automation.RegistroUsuarioTests"},{"name":"testMethod","value":"iniciarSesionConCredencialesValidas"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Proyecto-Demoblaze"},{"name":"subSuite","value":"automation.RegistroUsuarioTests"},{"name":"host","value":"Jeremias-Pc"},{"name":"thread","value":"9964@Jeremias-Pc.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Registro de usuario"}],"links":[],"name":"iniciarSesionConCredencialesValidas","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"El mensaje no es el mismo expected [Sign up successful.] but found [This user already exist.]","trace":"java.lang.AssertionError: El mensaje no es el mismo expected [Sign up successful.] but found [This user already exist.]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat automation.RegistroUsuarioTests.iniciarSesionConCredencialesValidas(RegistroUsuarioTests.java:45)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Caso de prueba T44 : Registrar un usuario con username y password valido","steps":[],"attachments":[],"parameters":[],"start":1734367548500,"stop":1734367557005}